frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -121,4 +281,20 @@ export type UninstallOperatorModalProps = {
   csv?: ClusterServiceVersionKind;
 };
 
+export type OperandsProps = {
+  csvName: string;
+  operands: K8sResourceKind[];
+  operandsLoaded: boolean;
+  deleteAllOperands: boolean;
+  setDeleteAllOperands: (tf: boolean) => void;
+  inFlight?: boolean;
+  cancel?: () => void;
+};
+
+export type OperandsTableProps = {
+  operands: K8sResourceKind[];
+  loaded: boolean;
+  customData: any;
```

Comment:
`customData` is passed thru to every row and utilized by our `Row` method in the operand's `<Table ...>` impl., however we are considering whether we should use the full resource table component in this context. It's not really meant for a dialog.

Reply:
Updated to include some verbiage about 'operand instances':
![image](https://user-images.githubusercontent.com/12733153/121247964-d4fa5500-c870-11eb-8032-9707c92d6941.png)

TBH, I don't know what the text "If your Operator deployed applications on the cluster or configured off-cluster resources, these will continue to run and require manual cleanup." is refering too?  @TheRealJon ?

Also, please see [recent changes](https://github.com/openshift/console/pull/8895) to the operator uninstall modal to also remove plug-ins.  This will add the following verbiage to the end of text if user has privleges to remove them:  "... The console plugin provided by this operator will be disabled and removed. "

Reply:
Replaced with simple inline table, now passing `csvName` and `cancel` directly.